### PR TITLE
Untitled Pull Request

### PR DESCRIPTION
**Pull request by cybershen**
Test

### CODE REVIEW

**Comment by cybershen** [comment]
HELLLO

File: PVRedactor/PVRedactor/ContentView.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        VStack {
+            Image(systemName: "globe")
+                .imageScale(.large)
+                .foregroundColor(.accentColor)
+            Text("Hello, world!")
+            Text("addasdasd")
```
Comment (SerhiiKopytchyk) [inline PVRedactor/PVRedactor/ContentView.swift:17]: Check spelling

**Comment by SerhiiKopytchyk** [comment]
New comment

File: PVRedactor/PVRedactor/ContentView.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+//
+//  Created by Serhii Kopytchuk on 27.12.2022.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        VStack {
+            Image(systemName: "globe")
```
Comment (SerhiiKopytchyk) [inline PVRedactor/PVRedactor/ContentView.swift:13]: Change image

**Review by SerhiiKopytchyk — CHANGES_REQUESTED** [review]

**Review by SerhiiKopytchyk — APPROVED** [review]

**Pull request merged**